core_backend/tests/rails/data/language_identification.yaml

```diff
@@ -0,0 +1,58 @@
+# These were generated using google translate but we may want to
+# improve this with a native speaker. These might be too "pure".
+
+HAUSA:
+  - Ina da yara biyu masu hanci
+  - Jiya ina jin barci akan kujera yau kuma bayana yayi zafi
+  - Shin ya zama al'ada a gare ku don jin zafi a duk lokacin da kuka yi atishawa?
+  - Menene wannan?
+  - Sannun ku da zuwa #h/t: Fola from here on
+  - Ni yarinya ne
+  - Zo ka chi abunchi
+  - Ina kwana Maman mu
+  - Wannan shago na ne
+ENGLISH:
+  - I have two children. You see I girl, what is the probability the other is also a girl?
+  - No idea
+  - Why you say that?
+XHOSA:
+  - Umama ngugqirha
+  - Utata ngumongikazi
+  - Ukuba intamo yam yayifuna ukwenza oko?
+  - Iintsana zikhala kakhulu, huh?
+YORUBA: #h/t: Fola
+  - Ni bo ló ti ri owo yen?
+  - Eyin melo ni e wa ni be?
+  - Ki ni itumo oruko ẹ?
+  - Ki ni o jẹ lánà?
+  - Omo Ibadan ni mi
+IGBO: #h/t: Fola
+  - agụụ na-agụ m
+  - agam aga ahia echi
+  - ị hụla ngozi? ana m achọ ya.
```

Comment:
In my run I got Zulu for this sentence! :/

Reply:
Really? So weird. We may have to use a different - more stable / more robust model than GPT for language detection

Reply:
It passes for me! Probably gpt being weird :/ We should use a better model for this in the future.

Reply:
I tried it again and i got this. Going to leave it as it's a nice test case for where gpt fails.